From c9e491bf48183a9fa7ab20518ffb780aed177df1 Mon Sep 17 00:00:00 2001
From: Roman Stratiienko <r.stratiienko@gmail.com>
Date: Sat, 26 Aug 2023 12:48:05 +0300
Subject: [PATCH 16/16] README

This series brings 2 additional backends to minigbm:
1. Generic (gbm_mesa). It works similarly to gbm_gralloc but has the benefits
   of using minigbm core logic/helpers to allocate more advanced format
   combinations.

2. Lightweight (dma_heaps). It requires a full description for every platform.
   Therefore, at the moment, it is only experimental.

For more information, reviews, comments, suggestions and questions visit:
https://github.com/GloDroid/glodroid_forks/pull/10

Change-Id: I2c3470d79c385b2b2e7a376f4ba6438f81d5c269
---
 _aospext_patched | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 _aospext_patched

diff --git a/_aospext_patched b/_aospext_patched
new file mode 100644
index 0000000..e69de29
-- 
2.43.0

